### ISSUE TITLE
why isn't cvv or zip code required?

### ISSUE THREAD

**Issue by faceyspacey** [alawong/react-native-braintree#22]
or at least made available for deeper authentication by Braintree (as they will verify they match if provided)?

I'm referring primarily to `getCardNonce`.

**Comment by ferndopolis** [comment]
yeah the code for getCardNonce was intended for Braintree SDK v3.9 which did not support CCV
now its on v4.0 i believe so we need to update the getCardNonce method. 

in meantime you can check out https://github.com/ferndopolis/react-native-braintree-card  this one is on v4.0

i'll will submit a PR

**Comment by faceyspacey** [comment]
there's also the address info, which you can verify against as well.

**Comment by ferndopolis** [comment]
so getCardNonce currently bridges the `initWithNumber` method which only accepts 
the following parameters: creditCardNumber, expirationMonth, expirationYear, cvv

I could add the `initWithParameters` method just check if argument above or object is passed.  

http://cocoadocs.org/docsets/Braintree/4.7.0/Classes/BTCard.html
